CondFormats/HcalObjects/interface/HcalDetIdRelationship.h

```diff
@@ -0,0 +1,18 @@
+#ifndef HcalDetIdRelationship_h
+#define HcalDetIdRelationship_h
+
+#include "DataFormats/HcalDetId/interface/HcalDetId.h"
+#include "DataFormats/HcalDetId/interface/HcalOtherDetId.h"
+#include "DataFormats/HcalDetId/interface/HcalCastorDetId.h"
+#include "DataFormats/HcalDetId/interface/HcalZDCDetId.h"
+#include <vector>
+
+template<class Item> bool hcalEqualDetId(Item* cell, DetId fId) {
```

Comment:
@bsunanda can the pointer to `Item` template class become `const` (the values are simply retrieved), and the `DetId` become a `const` reference (`fId` is simply accessed, not modified,and we avoid a copy)?
If you want this to enter the next pre-release, we can make another PR later.